src/plugins/media/nodeviews/mediaSingle.tsx

```diff
@@ -44,8 +44,7 @@ import {
   floatingLayouts,
   isRichMediaInsideOfBlockNode
 } from '@atlaskit/editor-core/utils/rich-media-utils'
-import { getAttrsFromUrl } from '@atlaskit/media-client'
-import { isMediaBlobUrlFromAttrs } from '../utils/media-common'
+import { cozyMediaOptions } from '../../../config/cozy-media-options'
```

Comment:
sometimes you import from : 
```
import { cozyMediaOptions } from 'config/cozy-media-options'
```

you should harmonize

Reply:
Yes vscode didn't seem very consistant here I used the default choice each time. Will watch out from now on. updated